js/plugins/vertexai/src/index.ts

```diff
@@ -111,24 +110,25 @@ export {
   type GeminiVersionString,
 };
 
-async function initializer(ai: Genkit, options?: PluginOptions) {
+async function initializer(options?: PluginOptions) {
   const { projectId, location, vertexClientFactory, authClient } =
     await getDerivedParams(options);
 
+  const actions: ResolvableAction[] = [];
+
   Object.keys(SUPPORTED_IMAGEN_MODELS).map((name) =>
```

Comment:
I think i'd prefer a `for ... of` loop here instead of a map, now that we're doing something with it.